mkosi/manifest.py

```diff
@@ -234,7 +234,21 @@ def has_data(self) -> bool:
         return len(self.packages) > 0
 
     def as_dict(self) -> Dict[str, Any]:
+        image = {
+            "distribution": self.args.distribution.name,
+            "release": self.args.release,
+            "architecture": self.args.architecture,
+            "format": self.args.output_format.name,
+        }
+        if self.args.image_id is not None:
+            image["name"] = self.args.image_id
+        else:
+            image["name"] = "image"
+        if self.args.image_version is not None:
+            image["version"] = self.args.image_version
```

Comment:
Could we maybe also set this to some fallback value? This would make it easier for users, since they can assume keys are just there.

Reply:
Checking for a key existence when parsing json is easy. If there's no version then we don't use it anywhere in the image, so we shouldn't make one up for the manifest. I intend to use the manifest content to make decisions about the image itself.